.github/workflows/end2end.yaml

```diff
@@ -445,6 +445,16 @@ jobs:
       - name: Debug wait
         uses: ./.github/actions/debug-wait
         timeout-minutes: 60
+      - name: Upload results
+        if: "!cancelled() && env.TRUNK_TOKEN"
+        uses: trunk-io/analytics-uploader@main
```

Comment:
(same for each)

Reply:
This will trigger a syntax error:
<img width="620" alt="Screenshot 2024-04-23 at 1 39 44 PM" src="https://github.com/scality/Zenko/assets/8408330/d499beda-6948-41c8-a699-ef9395edd9dd">